sbin/build.sh

```diff
@@ -130,6 +130,7 @@ buildingTheRestOfTheConfigParameters()
   CONFIGURE_ARGS="${CONFIGURE_ARGS} --with-jvm-variants=${JVM_VARIANT}"
   CONFIGURE_ARGS="${CONFIGURE_ARGS} --with-cacerts-file=${WORKING_DIR}/cacerts_area/security/cacerts"
   CONFIGURE_ARGS="${CONFIGURE_ARGS} --with-alsa=${WORKING_DIR}/alsa-lib-${ALSA_LIB_VERSION}"
+  CONFIGURE_ARGS="${CONFIGURE_ARGS} --with-freemarker-jar=${WORKING_DIR}/freemarker-${FREEMARKER_LIB_VERSION}/lib/freemarker.jar"
```

Comment:
Can this clash with the freetype if building the 'regular' OpenJDK?  I assume not.

Reply:
I think that configure normally ignores unknown arguments, for safety though we should wrap this in an if statement to only use if we're building openj9

Reply:
Yes, that's a good point. Thanks for the feedback, I'll add an if statement.